ag/enrollment.go

```diff
@@ -20,7 +26,13 @@ func (m *Enrollment) UpdateSlipDays(start time.Time, assignment *Assignment, sub
 	// if score is less than limit and it's not yet approved, update slip days if deadline has passed
 	if submission.Score < assignment.ScoreLimit && submission.Status != Submission_APPROVED && sinceDeadline > 0 {
 		// deadline exceeded; calculate used slipdays for this assignment
-		m.updateSlipDays(assignment.GetID(), uint32(sinceDeadline/days))
+		slpDays, slpHours := uint32(sinceDeadline/days), sinceDeadline%days
```

Comment:
Could you please add a few tests for this feature? There are some tests in `slipdays_test.go` that can be expanded with tests for grace period.

Reply:
PS: The logic Looks good to me! But it is always good to have tests for this stuff.